lib/kafka_ex/server.ex

```diff
@@ -127,11 +127,14 @@ defmodule KafkaEx.Server do
   end
 
   def handle_call({:create_stream, handler}, _from, state) do
-    state = unless state.event_pid && Process.alive?(state.event_pid) do
+    if state.event_pid && Process.alive?(state.event_pid) do
+      info = Process.info(self)
+      Logger.log(:warn, "#{info[:registered_name]} already streaming")
```

Comment:
I would still include the handler in the log msg because the error can come from the default process which can be confusing